freqtrade/exchange/ftx.py

```diff
@@ -28,6 +28,7 @@ class Ftx(Exchange):
         # (TradingMode.MARGIN, Collateral.CROSS),  # TODO-lev: Uncomment once supported
         # (TradingMode.FUTURES, Collateral.CROSS)  # TODO-lev: Uncomment once supported
     ]
+    mark_ohlcv_price = 'index'
```

Comment:
I think this might be something to be moved into `_ft_has`
it's an exchange-specific attribute ... and i think it's better to have all of these combined in one sub-structure (that also allows easier testing of non-officially supported exchanges ...)

Reply:
Another question - won't we need both this and #5780 for proper dry-run (Assuming we support "shutdown" as well (so we need to catch-up)?

now obviously, we can do these 1 after the other (no support for shutdown for the moment) - it's more an understanding question than anything else

Reply:
> Another question - won't we need both this and #5780 for proper dry-run (Assuming we support "shutdown" as well (so we need to catch-up)?

No, #5780 is just supposed to be an addition to download candles with the same methods.

The amount of candles needed for dry run is so small because it's on the 1h, that if it were shut down for a week, that would be 168/pair, that didn't seem like an amount of data that was worth optimizing or saving, and that it could just be download on the spot

Reply:
i could also limit it to just every 4h, since ftx isn't available now anyway, and every other exchange that I've looked at (just the ft classes right now) isn't more common than this

Reply:
my point is - if we reuse the methods for downloading historic data - why don't we reuse these methods also for this downloads?
otherwise we'll have the same code (the API call is the same, except for the "start" date) twice ...

Reply:
Yeah I guess we could try to reuse some of those methods, I haven't really analyzed them in depth, I just wanted to make it easy to download the mark and index price candles using them